tests/test_random.py

```diff
@@ -0,0 +1,62 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import retworkx
+
+
+class TestGNPRandomGraph(unittest.TestCase):
+
+    def test_random_gnp_directed(self):
+        graph = retworkx.directed_gnp_random_graph(20, .5, 10)
+        self.assertEqual(len(graph), 20)
+        self.assertEqual(len(graph.edges()), 104)
```

Comment:
Correct me if im wrong, but if the number of edges is random how can this reproduce the same number without setting a random seed? Is there a seed set somewhere else?

Reply:
Yeah, the seed is 10 it's the 3rd arg on the random call in L21, I just passed the kwarg in positionally. I'll add the name to avoid confusion.

Reply:
Oh right! Thanks 👍

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/94/commits/ec119572439c08a284a03b2f8315320ace9d57ff